docs/reference/modules/node.asciidoc

```diff
@@ -360,10 +360,13 @@ To create a dedicated {ml} node, set:
 node.roles: [ ml, remote_cluster_client] <1>
 xpack.ml.enabled: true <2>
 ----
-<1> The `remote_cluster_client` role is optional but strongly recommended.
+<1> The `remote_cluster_client` role is optional but strongly recommended on all {ml} nodes.
```

Comment:
Since this section is about ML nodes, I don't think this clause is necessary.
```suggestion
<1> The `remote_cluster_client` role is optional but strongly recommended.
```